Autocoders/Python/bin/codegen.py

```diff
@@ -1208,7 +1208,14 @@ def main():
 
     # Configure the logging.
     log_level = opt.logger.upper()
-    log_level_dict = {}
+    log_level_dict = {
+        "QUIET": None,
+        "DEBUG": logging.DEBUG,
+        "INFO": logging.INFO,
+        "WARNING": logging.WARN,
+        "ERROR": logging.ERROR,
+        "CRITICAL": logging.CRITICAL,
+    }
 
     log_level_dict["QUIET"] = None
```

Comment:
I think this code was to be deleted?  You moved it up, but I don't see the removal of the old-form.

Reply:
Oops I didn't see that I left the old form just below... I was on another PR at the same time. I'll fix that later tonight.